### PR TITLE
Fix AMD overdrive 8

### PR DESCRIPTION
**Pull request by PhyxionNL**

### CODE REVIEW

File: LibreHardwareMonitorLib/Hardware/Gpu/AmdGpu.cs [hunk 1]
```diff
@@ -137,9 +137,9 @@ public AmdGpu(AtiAdlxx.ADLAdapterInfo adapterInfo, ISettings settings)
                 }
             }
```
Comment (Rem0o) [inline LibreHardwareMonitorLib/Hardware/Gpu/AmdGpu.cs:139]: Hard-coded "1", really?

**Comment by kolos** [comment]
This removes (Voltages/Clocks/Temperature) reading for me:

image

I have this in the report:
```
AMD GPU

AdapterIndex: 0

Overdrive Caps

 Status: ADL_ERR_NOT_SUPPORTED
 Supported: 0
 Enabled: 0
 Version: 7

Overdrive5 Parameters

 Status: ADL_OK
 NumberOfPerformanceLevels: 8
 ActivityReportingSupported: 1
 DiscretePerformanceLevels: 1
 EngineClock.Min: 10500
 EngineClock.Max: 125100
 EngineClock.Step: 500
 MemoryClock.Min: 0
 MemoryClock.Max: 160000
 MemoryClock.Step: 500
 Vddc.Min: 0
 Vddc.Max: 0
 Vddc.Step: 0

Overdrive5 Temperature

 Status: -100
 Value: 0

Overdrive5 FanSpeed

 Status RPM: -100
 Value RPM: 0
 Status Percent: -100
 Value Percent: 0

Overdrive5 CurrentActivity

 Status: ADL_OK
 EngineClock: 400
 MemoryClock: 1600
 Vddc: 0,001
 ActivityPercent: 1600
 CurrentPerformanceLevel: 0
 CurrentBusSpeed: 2500
 CurrentBusLanes: 16
 MaximumBusLanes: 16
```

Suggesting moving out ```ADL2_New_QueryPMLogData_Get``` from ```if (_overdriveApiSupported)``` to always poll it.

**Comment by PhyxionNL** [comment]
That's strange, as ADL2_New_QueryPMLogData is OD8 and should report it as such. Can you try the latest build?

**Comment by kolos** [comment]
Strange indeed..

Anyways, I got my sensors back:
image

I'm not sure that we need ```if (_currentOverdriveApiLevel >= 8 || !_overdriveApiSupported)``` there, since ```if (AtiAdlxx.ADL2_New_QueryPMLogData_Get(_context, _adapterIndex, ref logDataOutput) == AtiAdlxx.ADLStatus.ADL_OK)``` checks anyways.

**Comment by Rem0o** [comment]
Seems like some AMD GPU support issues persists, see the logs in:
 https://github.com/Rem0o/FanControl.Releases/issues/264#issuecomment-863606595

**Pull request merged**